libs/vips/Makefile

```diff
@@ -34,38 +36,35 @@ define Package/vips
   DEPENDS:=+glib2 +libexif +libexpat +libjpeg +libpng +giflib +libxml2 +libstdcpp $(INTL_DEPENDS)
```

Comment:
Wonder if libstdcpp is needed. intl_depends comes with glib2.

Reply:
Removed both $(INTL_DEPENDS) and +libstdcpp.